test/api/ApolloGraphClientTest.ts

```diff
@@ -16,71 +15,6 @@ import {
 
 describe("ApolloGraphClient", () => {
 
-    describe("deprecated api", () => {
```

Comment:
I didn't notice this before, but this was smart to do.

Reply:
I don't think I did that. Not that clear. ;-)